regcore_read/views/es_search.py

```diff
@@ -37,6 +39,10 @@ def search(request, doc_type):
             term['version'] = version
         if regulation:
             term['regulation'] = regulation
+        if is_root:
+            term['is_root'] = is_root
+        if is_subpart:
+            term['is_subpart'] = is_subpart
```

Comment:
Meanwhile, I didn't test the non-haystack elastic backend manually. I'll do that and check whether this is right--although I'd still like to drop one backend or the other and stop maintaining unused code.